analysis/verify_pdf_coverage.py

```
@@ -0,0 +1,275 @@
+        self.pdf_base_dir = Path(pdf_base_dir)
+        
+    def close(self):
+        """Close database connection"""
+        self.conn.close()
+        
+    def normalize_arxiv_id_to_path(self, arxiv_id):
+        """Convert arXiv ID to expected file path"""
+        # Remove version suffix if present
+        if 'v' in arxiv_id:
+            arxiv_id = arxiv_id.split('v')[0]
+            
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)